python/lsst/afw/table/baseColumnView/baseColumnView.cc

```diff
@@ -94,12 +81,26 @@ static void declareBaseColumnView(py::module &mod) {
     declareBaseColumnViewFlagOverloads(cls);
     // std::string columns are not supported, because numpy string arrays
     // do not have the same memory model as ours.
-    declareBaseColumnViewOverloads<lsst::afw::geom::Angle>(cls);
     declareBaseColumnViewArrayOverloads<std::uint8_t>(cls);
     declareBaseColumnViewArrayOverloads<std::uint16_t>(cls);
     declareBaseColumnViewArrayOverloads<int>(cls);
     declareBaseColumnViewArrayOverloads<float>(cls);
     declareBaseColumnViewArrayOverloads<double>(cls);
+    // Angle requires custom wrappers, because ndarray doesn't
+    // recognize it natively; we just return a double view
+    // (e.g. radians).
```

Comment:
This is breaking the encapsulation of `Angle` a bit right? Or does the API for `getData` specify that the values are in radians?

Reply:
Yes, it's breaking encapsulation, but it was broken before; I'm just breaking it differently now.